_config.yml

```diff
@@ -19,4 +19,5 @@ markdown: kramdown
 # This is used in the Atom & RSS feeds and in the sitemap
 production_url: "http://nunit.org"
 rss_path: /rss.xml
-atom_path: /atom.xml
\ No newline at end of file
+atom_path: /atom.xml
+host: 0.0.0.0
```

Comment:
Need to listen on all addresses so that we can forward the port from the Docker container to the host